### PR TITLE
Re-render the header once, to populate the title and author

### LINKED ISSUE #444: Title of Procedure is not updating properly when returning to it

### PR DESCRIPTION
**Pull request by Kapin**
Closes #444

### CODE REVIEW

File: src-backbone/app/js/views/builder/builderHeaderView.js [hunk 1]
```diff
@@ -32,6 +37,8 @@ module.exports = Marionette.ItemView.extend({
         });
     },
 
+    _renderOnce: _.once(function() { this.render(); }),
```
Comment (connorcimowsky) [inline src-backbone/app/js/views/builder/builderHeaderView.js]: Would prefer the function parameter to be on multiple lines.

**Comment by connorcimowsky** [comment]
LGTM once @Trinovantes weighs in on whether this will fix the race condition once and for all

File: src-backbone/app/js/views/builder/builderHeaderView.js (hunk 1 above)
Comment (Kapin) [inline src-backbone/app/js/views/builder/builderHeaderView.js]: can do

**Comment by Trinovantes** [comment]
probably +1

**Pull request merged**